cartoframes/data/observatory/category.py

```diff
@@ -0,0 +1,57 @@
+import pandas as pd
+
+from cartoframes.data.observatory.repository.category_repo import get_category_repo
+from cartoframes.data.observatory.repository.dataset_repo import get_dataset_repo
```

Comment:
Since we want the API methods to be static we can't have the repositories instances in these classes, that's why I created those helper methods, but I'm not sure if I like it xD

Would be better to import the Repo instead and create a private instance for the module? Something like:

```
from cartoframes.data.catalog.repository.category_repo import CategoryRepository

 __category_repo = CategoryRepository()
```

Reply:
Since we are going to use the Repository pattern to switch to a different backend API when needed, what of these options would be easier to replace when we have a different backend API?

Reply:
I prefer the helper. Less code and less coupling.

> what of these options would be easier to replace when we have a different backend API?

I think, we could change to a different backend just changing the `RepoClient`.